src/System.Numerics.Vectors/src/System.Numerics.Vectors.csproj

```diff
@@ -41,43 +36,40 @@
       <DesignTime>True</DesignTime>
       <DependentUpon>Vector.tt</DependentUpon>
     </Compile>
+    <Compile Include="System\Numerics\Vector_Operations.cs" />
+  </ItemGroup>
+  <!-- Portable version only -->
+  <ItemGroup Condition="'$(TargetGroup)'==''">
+    <Compile Include="System\Numerics\Matrix3x2.cs" />
+    <Compile Include="System\Numerics\Matrix4x4.cs" />
+    <Compile Include="System\Numerics\Plane.cs" />
+    <Compile Include="System\Numerics\Quaternion.cs" />
     <Compile Include="System\Numerics\Vector2.cs" />
     <Compile Include="System\Numerics\Vector2_Intrinsics.cs" />
     <Compile Include="System\Numerics\Vector3.cs" />
     <Compile Include="System\Numerics\Vector3_Intrinsics.cs" />
     <Compile Include="System\Numerics\Vector4.cs" />
     <Compile Include="System\Numerics\Vector4_Intrinsics.cs" />
-    <Compile Include="System\Numerics\Vector_Operations.cs" />
   </ItemGroup>
-  
   <ItemGroup Condition="'$(TargetGroup)'=='net46'">
     <TargetingPackReference Include="mscorlib" />
     <TargetingPackReference Include="System.Numerics" />
-    <Compile Include="System\Numerics\Vector.cs" />
-    <Compile Include="System\Numerics\Vector_Operations.cs" />
-    <Compile Include="System\Numerics\ConstantHelper.cs" />
-    <Compile Include="System\Numerics\JitIntrinsicAttribute.cs" />
-    <Compile Include="System\Numerics\Register.cs" />
-    <Compile Include="System\Numerics\HashCodeHelper.cs" />
   </ItemGroup>
   <ItemGroup>
-    <None Include="project.json" />
-    <None Include="System\Numerics\ConstantHelper.tt">
+    <None Include="System\Numerics\GenerationConfig.ttinclude" />
```

Comment:
I had VS add the other .tt files to the project because they actually produce something (they have this special "Generator" property on them), but GenerationConfig.ttinclude isn't actually used to directly produce anything as it's just used by the other .tt files. I think the others need to be Content files so that VS knows to run the T4 templating tool on them.